doc/release-notes-5158.md

```diff
@@ -0,0 +1,4 @@
+Updated RPCs
+--------
+
+- `quorum info`: New field `previouslyFailedConsecutiveDKGs` will be returned for rotated LLMQs. This field will hold the number of previously consecutive failed DGKs for the corresponding quorumIndex before the currently active one. Note: If no previously commitments were found then 0 will be returned for `previouslyFailedConsecutiveDKGs`.
```

Comment:
```suggestion
- `quorum info`: The new `previouslyFailedConsecutiveDKGs` field will be returned for rotated LLMQs. This field will hold the number of previous consecutive DKG failures for the corresponding quorumIndex before the currently active one. Note: If no previous commitments were found then 0 will be returned for `previouslyFailedConsecutiveDKGs`.
```

Also, what do you think about "previousConsecutiveDKGFailures" as the field name? Might sound a little better :man_shrugging: